apps/health_campaign_field_worker_app/lib/widgets/network_manager_provider_wrapper.dart

```diff
@@ -14,7 +14,7 @@ import '../blocs/app_initialization/app_initialization.dart';
 import '../data/data_repository.dart';
 import '../data/local_store/sql_store/sql_store.dart';
 import '../data/network_manager.dart';
-import '../data/repositories/local/address.dart';
+import '../data/repositories/local/adverse_event.dart';
```

Comment:
why we renamed from address to adverse_event

Reply:
Its ununsed import